pageserver/src/rocksdb_storage.rs

```diff
@@ -30,11 +31,35 @@ impl StorageKey {
     }
 }
 
+struct GarbageCollector {
```

Comment:
Please add comments to explain how this works.

Reply:
Done